src/engine/server/sv_ccmds.cpp

```diff
@@ -114,15 +114,6 @@ class MapCmd: public Cmd::StaticCmd {
 static MapCmd MapCmdRegistration("map", "starts a new map", false);
 static MapCmd DevmapCmdRegistration("devmap", "starts a new map with cheats enabled", true);
 
-void MSG_PrioritiseEntitystateFields();
-void MSG_PrioritisePlayerStateFields();
-
-static void SV_FieldInfo_f()
```

Comment:
The netcode sends the deltas for each field in the table up to the last one that had any changes. The idea of this command is to run the game for a while and then print the fields in order from most to least frequently changed, to find an order which would require transmitting the fewest fields per packet.